### ISSUE TITLE
Why am I getting no bounding boxes when predicting?

### ISSUE THREAD

**Issue by Taha0229** [ultralytics/ultralytics#8314]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

I followed the documentation to build a very simple object detection model, but while inferencing I am not getting any bounding boxes. I can see bounding boxes on the batches of images on which the training of done. Also, no bounding boxes are created for validation images. Please help-

Here's my code-
`from google.colab import drive
import os

drive.mount('/content/drive')

import zipfile

zip_ref = zipfile.ZipFile("/content/drive/MyDrive/yolo_testing_data.zip", 'r')
zip_ref.extractall("/content")
zip_ref.close()

!pip install ultralytics

from ultralytics import YOLO

model = YOLO('yolov8n.pt')

results = model.train(data='./data.yaml', epochs=5, imgsz=640, project="./output", dropout=0.2) 

metrics = model.val()

model = YOLO('./output/train10/weights/best.pt')
results = model.predict(source='./datasets/data/train/images/100.jpeg', conf=0.25)

results[0].boxes
`

![image](https://github.com/ultralytics/ultralytics/assets/113607983/29ea5666-5241-4b79-848b-45900ab5698a)

training batch images-
![1](https://github.com/ultralytics/ultralytics/assets/113607983/6e3bfe4f-7473-4a2d-83a1-b4f1d6f3a219)

val batch images-
![val2](https://github.com/ultralytics/ultralytics/assets/113607983/20123476-c748-458a-8274-a5e8ac304f88)

**Comment by glenn-jocher** [comment]
@Taha0229 hi there! 👋 It looks like you've done a great job following the documentation for setting up a simple object detection model. If you're not seeing any bounding boxes during inference, here are a couple of things you might want to check:

1. **Confidence Threshold**: Ensure that the confidence threshold is not set too high, which might be filtering out valid detections. You can try lowering it to see if you get any bounding boxes.

2. **Model Weights**: Confirm that the model weights loaded for prediction are indeed the best ones from training. Sometimes, if the training hasn't converged well, the weights might not be optimal for detection.

3. **Data Normalization**: Make sure that the input images for prediction are preprocessed in the same way as during training (e.g., normalization).

4. **Annotations**: Verify that the annotations in your training and validation datasets are correct and consistent.

Here's a small code snippet to visualize the predictions with a lower confidence threshold:

```python
results = model.predict(source='./datasets/data/train/images/100.jpeg', conf=0.1)
results.show()
```

If you're still facing issues, feel free to share more details like the training loss curves or any warnings that might have appeared during training. We're here to help! 😊

**Comment by purv2100** [comment]
Hello,
I am trying to do an image classification task using the standard dataset format for the classification task which does not include any labels file, just the images. For this when I try to get an inference on the image it gives me null for Result.Boxes. I understand it is an image classification task but I would also like the result to give me a bounding box if possible. What should I change to get the bounding box in the result.Boxes classes?
